public/index.html

```diff
@@ -2,14 +2,18 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="/map-marked-alt-solid.svg" />
+    <script
+      src="https://kit.fontawesome.com/419f457776.js"
+      crossorigin="anonymous"
+    ></script>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    <meta name="theme-color" content="#5B1B52" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="App To Show You Wikipedia Entry Of What is Nearby"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="apple-touch-icon" href="map-marked-alt-solid.png" />
```

Comment:
For your `<link rel="icon"/>`, you included the leading `/`, so consider keeping it consistent between the 2 `href` values.

Is there a reason you removed the `%PUBLIC_URL%` variable from the path?

Reply:
I wish it were possible to leave comments on binary files in GitHub, but anyway....

`public/map-marked-alt-solid.png` is bigger than you need. Consider shrinking it down to 512px at the biggest side and increasing the other side to be 512px as well (i.e. make it square).

Reply:
Thank you for the feedback, Andrew.  I mistakenly removed the `%PUBLIC_URL%` variable when I was adding the icon paths.  I've also resized the png and made it square.

Thanks to @alxmcr's mention of the icon path name, I renamed the icon and image file to a more clear and readable `map-icon` which I think will help us down the road.